tests/vmi_configuration_test.go

```diff
@@ -2559,6 +2559,26 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				pinnedCPUsList, err := hw_utils.ParseCPUSetLine(output, 100)
 				Expect(err).ToNot(HaveOccurred())
 
+				output, err = tests.ListCgroupThreads(readyPod)
+				Expect(err).ToNot(HaveOccurred())
+				pids := strings.Split(output, "\n")
+
+				getProcessNameErrors := 0
+				By("Expecting only vcpu threads on root of pod cgroup")
+				for _, pid := range pids {
+					if len(pid) == 0 {
+						continue
+					}
+					output, err = tests.GetProcessName(readyPod, pid)
+					if err != nil {
+						getProcessNameErrors++
+						continue
+					}
+					Expect(output).To(ContainSubstring("CPU "))
+					Expect(output).To(ContainSubstring("KVM"))
+				}
+				Expect(getProcessNameErrors).Should(BeNumerically("<=", 1))
```

Comment:
Please add a brief explanation to your [discussion above](https://github.com/kubevirt/kubevirt/pull/8233/files#r982376550) so people would have context to why this is needed. Maybe something like:
```go
Expect(getProcessNameErrors).Should(BeNumerically("<=", 1), `only one "cat" process should provide an error since it spawns on the VM's pod cgroup`)
```